### PR TITLE
Change dimension upon loading an Image through UI

### LINKED ISSUE #1391: (displayed) Image dimensions do not change upon loading a new image
### Please describe the problem (or idea)
Screenshot from 2019-12-25 00-33-09
Screenshot from 2019-12-25 00-33-15

As can be seen in the above two screenshots, the image dimensions did not change even though a different, rotated image was loaded.

### Please show us where to look

https://beta.sequencer.publiclab.org

### What's your PublicLab.org username?

> This can help us diagnose the issue:

### Browser, version, and operating system

> Many bugs are related to these -- please help us track it down and reproduce what you're seeing!

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site.

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

### PR DESCRIPTION
**Pull request by ataata107**
Fixes #1391 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [X] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [X] code is in uniquely-named feature branch and has no merge conflicts
* [X] PR is descriptively titled
* [X] ask `@publiclab/is-reviewers` for help, in a comment below
* [X] Dimension change is working as expected
> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software
Please make sure to get at least two reviews before asking for merging the PR as that would make the PR more reliable on our part
Thanks!

Screenshot from 2019-12-26 18-03-50
Screenshot from 2019-12-26 18-04-10

### CODE REVIEW

**Comment by ataata107** [comment]
publiclab/is-reviewers

**Comment by HarshKhandeparkar** [comment]
Does this do the reloading? Instead, I think there should be a load image callback or even which could trigger just the dimensions function(or anything else if you want). What do you think?

**Comment by ataata107** [comment]
Yes you’re correct 
By the way this doesn’t do reloading. This PR just triggers the dimension function as one loads an image.

**Comment by ataata107** [comment]
Sure

**Comment by ataata107** [comment]
@HarshKhandeparkar removed the comments and refactored please review

File: examples/lib/defaultHtmlStepUi.js [hunk 1]
```diff
@@ -353,6 +351,12 @@ function DefaultHtmlStepUi(_sequencer, options) {
     else $step('.wasm-tooltip').fadeOut();
   }
 
+  function changeDimension(step){
```
Comment (HarshKhandeparkar) [inline examples/lib/defaultHtmlStepUi.js]: Can this function be called `updateDimensions`? The current name may be misleading. Please add JSDoc description comment for the function too. Thanks!

File: examples/lib/defaultHtmlStepUi.js [hunk 2]
```diff
@@ -352,6 +350,17 @@ function DefaultHtmlStepUi(_sequencer, options) {
     }
     else $step('.wasm-tooltip').fadeOut();
   }
+  /**
+   * Updates Dimension of the image
```
Comment (HarshKhandeparkar) [inline examples/lib/defaultHtmlStepUi.js]: `@description Updates...` would be better.

**Comment by jywarren** [comment]
🎉

**Pull request merged**